routes.go

```diff
@@ -209,12 +209,13 @@ func (s *Server) handleEvent() http.HandlerFunc {
 				formatJSONError(w, http.StatusInternalServerError, err.Error())
 				return
 			}
-			// Marshal event for production to kafka topic
-			message, err := json.Marshal(&e)
-			if err != nil {
-				log.Error("Failed to unmarshal event JSON for message queue")
+			{
+				data, err := json.Marshal(&e)
```

Comment:
This could be the original data POSTed by the client. Nothing special happens to the data by being inserted into the database.

Reply:
This means that the JSON struct field tags on `type event` are not necessary either.